src/lib/Libifl/auth.c

```diff
@@ -339,17 +332,14 @@ load_auths(int mode)
 void
 unload_auths(void)
 {
-	int i = 0;
-
-	if (auths == NULL)
+	if (loaded_auths == NULL)
```

Comment:
you don't really need this if condition, the while loop won't be executed anyways if it is NULL